### PR TITLE
Testing .travis.yml

### PR DESCRIPTION
**Pull request by daisywatson**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+language: node_js
+node_js:
+  - "8"
+  
+addons:
+apt:
+  sources:
+    - google-chrome
+  packages:
+    - google-chrome-stable
```
Comment (mappuji) [inline .travis.yml]: dont use chrome, use phantom js, if we just confirm the functionalities.

http://karma-runner.github.io/0.8/plus/Travis-CI.html

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  - TRAVIS_NODE_VERSION="7.9.0"
+cache:
+   directories:
+     - $(npm root -g)
+     - node_modules
+
+notifications:
+  webhooks:
+    urls:
+      - https://webhooks.gitter.im/e/063276d3941fa8443947
```
Comment (mappuji) [inline .travis.yml]: I think we can use private environment variable and hiding our notifications link. We don't want some people spamming our channel right? https://docs.travis-ci.com/user/environment-variables/

Comment (dogi) [reply .travis.yml]: @mappuji thanks I think we should open an extra/new issue for all our `.travis.yml`es ;)

/me thinks we have at least 4 ...

File: .travis.yml (hunk 1 above)
Comment (paulbert) [reply .travis.yml]: As the tests are set up, they are using Chrome in headless mode so these are needed to run the tests.

**Pull request merged**